keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -449,30 +472,30 @@ status:
   - type: Conflict
     status: "True"
     lastTransitionTime: "2020-03-30T01:33:55Z"
-    message: "Conflicting type. Using \"ClusterSetIP\" from oldest service export in \"cluster-1\". 2/5 clusters disagree."
+    message: "Conflicting type. Using \"MultiClusterIP\" from oldest service export in \"cluster-1\". 2/5 clusters disagree."
```

Comment:
I think I'd rather see "MulticlusterServiceIP" here, which is aligned with the name of this concept, wdyt?